server/sched.go

```diff
@@ -192,6 +197,27 @@ func (s *Scheduler) processPending(ctx context.Context) {
 						break
 					}
 
+					// Block attempting to load a model larger than system memory + GPU memory
+					estimate := llm.EstimateGPULayers(gpus, ggml, pending.model.ProjectorPaths, pending.opts)
+					maxSize := systemMem.FreeMemory
+					for _, gpu := range gpus {
+						if gpu.Library == "cpu" {
+							continue
+						}
+						if loadedCount == 0 {
+							// If no other models are loaded, set the limit based on what's available
+							maxSize += gpu.FreeMemory
+						} else {
+							// Other models could be unloaded, favor total memory for limit
+							maxSize += gpu.TotalMemory
```

Comment:
Shouldn't these two be the same - i.e. if no models are loaded, `gpu.FreeMemory` == `gpu.TotalMemory`?

Reply:
If the system has a GUI, typically it's going to have other apps and the windowing system occupying some amount of VRAM on the primary GPU, which may be non-trivial.